src/services/processRenderer.tsx

```diff
@@ -31,6 +31,26 @@ export function renderProcessMessage(process: Process) {
   return <>{process.message.toString()}</>
 }
 
+export const renderProcessError = (step: Step, process: Process) => {
+  const errorMessage = process.errorMessage && (
+    <>
+      Error: {process.errorMessage.substring(0, 350)}
+      {process.errorMessage.length > 350 ? '...' : ''}
+    </>
+  )
+
+  const htmlMessage = process.htmlErrorMessage && (
```

Comment:
please see: https://github.com/lifinance/sdk/pull/22#discussion_r790546611

/n can be rendered with 
`
.display-linebreak {
  white-space: pre-line;
}
`